### PR TITLE
cgroups: reurn error when passing invalid argument to freezer

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by LK4D4** [comment]
@Mashimiao Tests fail :)

File: cgroups/fs/freezer.go [hunk 1]
```diff
@@ -41,6 +42,8 @@ func (s *FreezerGroup) Set(path string, cgroup *configs.Cgroup) error {
 			}
 			time.Sleep(1 * time.Millisecond)
 		}
+	default:
+		fmt.Errorf("Invalid argument %s to freezer.state", cgroup.Freezer)
 	}
 
 	return nil
```
Comment (LK4D4) [inline cgroups/fs/freezer.go]: You can remove it now. Because it's unreachable code.

Comment (Mashimiao) [inline cgroups/fs/freezer.go]: Hi LK4D4, Thanks for reviewing.
It seems it is.
But what I mainly want to do is to return error clearly when people pass  invalid argument accidentally.
I added new line testing configs.Undefined. How do you think about it?

**Comment by LK4D4** [comment]
LGTM
Also you have misspell in commit message `reurn`. You can fix if you want.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**